Sources/HummingbirdFoundation/Codable/JSON/JSONCoding.swift

```diff
@@ -17,7 +17,7 @@
 import Hummingbird
 import NIOFoundationCompat
 
-extension JSONEncoder: HBResponseEncoder {
+extension JSONEncoder: HBResponseEncoder, @unchecked Sendable {
```

Comment:
This is really shitty to have to do. Can we consider making our own `JSONDecoder`/Encoder that conforms to HBResponseEncoder, and conform that to `@unchecked Sendable`?

Reply:
Frustratingly JSONEncode/Decoder conforms to Sendable in Swift 5.9, latest macOS. Wrapping it would be a little crap. Allowing people to set the encoder to JSONEncoder straight is nice. Maybe I can do something at the HBApplication.Context level which is the reason it is required to be Sendable.

Reply:
Checked out the swift-foundation code and found this
```swift
extension JSONEncoder : @unchecked Sendable {}
```

Reply:
My worries are/were that the double-conformance might not compile down the line. Can we verify that